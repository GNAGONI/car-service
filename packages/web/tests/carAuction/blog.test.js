import React from 'react'
import renderer from 'react-test-renderer'
import Blog from '../../components/carAuction/blog'

it('renders correctly', () => {
  const tree = renderer.create(<Blog />).toJSON()
  expect(tree).toMatchSnapshot()
})
