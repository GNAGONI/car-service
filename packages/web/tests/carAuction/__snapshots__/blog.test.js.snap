// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="content bg-default posts-overlay section-blog"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        
      </div>
    </div>
  </div>
  <div
    className="content-holder"
  >
    <div
      className="posts-slideshow"
    >
      <div
        className="slick-slider stories-slider slick-initialized"
        dir="ltr"
      >
        <div
          className="btn-prev"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        />
        <div
          className="slick-list"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={null}
          onMouseMove={null}
          onMouseUp={[Function]}
          onTouchCancel={null}
          onTouchEnd={[Function]}
          onTouchMove={null}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "",
              }
            }
          >
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-3}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-2}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-1}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active slick-current"
              data-index={0}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={1}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={2}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={3}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={4}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={5}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={6}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={7}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <article
                  className="box-story post"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <div
                    className="box-holder post-holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <picture>
                        <source
                          media="(max-width: 767px)"
                          srcSet=",  2x\\""
                        />
                        <source
                          srcSet=",  2x\\""
                        />
                        <img
                          alt="Estimating the cost of fuel for a trip"
                          height="418"
                          src=""
                          width="380"
                        />
                      </picture>
                    </div>
                    <ul
                      className="meta"
                    >
                      <li>
                        <time
                          dateTime="2018-04-02"
                        >
                          02/04/18
                        </time>
                      </li>
                      <li>
                        <a
                          href="#"
                        >
                          Guide
                        </a>
                      </li>
                    </ul>
                    <div
                      className="caption"
                    >
                      <h2>
                        <a
                          href="#"
                        >
                          How to improve fuel efficiency
                        </a>
                      </h2>
                      <div
                        className="btn-block"
                      >
                        <a
                          className="read-more"
                          href="#"
                        >
                          Read more
                        </a>
                      </div>
                    </div>
                  </div>
                </article>
              </div>
            </div>
          </div>
        </div>
        <div
          className="btn-next"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        />
      </div>
    </div>
  </div>
</section>
`;
